plugins/in_http/http_prot.c

```diff
@@ -299,7 +299,7 @@ static int process_payload(struct flb_http *ctx, struct http_conn *conn,
         return -1;
     }
 
-    if (header->val.len == 16 &&
+    if (header->val.len >= 16 &&
```

Comment:
Technically this does the trick, should we add an inner check to ensure the seventeenth character is an allowed character or not? I think it wouldn't hurt and would probably do it but realistically speaking this is good enough.